lib/avrora/encoder.ex

```diff
@@ -136,4 +164,30 @@ defmodule Avrora.Encoder do
       end
     end
   end
+
+  @doc """
+  Encode binary Avro message with :plain format and given schema.
+
+  Retuns a plain Avro encoded message binary. This exists to mirror decode_plain/2,
+  which is necessary when decoding plain messages with a given schema due to the possibility of
+  a plain message starting with the magic byte, leading decode/2 to fail.
+
+  ## Examples
+
+      ...> payload = %{ "link_is_enabled" => false, "updated_at" => 1_586_632_500, "updated_by_id" => 1_00 }
+      ...> Avrora.Encoder.encode_plain(payload,"io.confluent.numeric_transfer")
+      {:ok, <<0, 232, 220, 144, 233, 11, 200, 1>>}
+  """
+  def encode_plain(payload, schema_name: schema_name) when is_map(payload) do
+    with {:ok, schema_name} <- Name.parse(schema_name) do
+      unless is_nil(schema_name.version) do
+        Logger.warn(
+          "encoding message with schema version is not supported yet, `#{schema_name.name}` used instead"
+        )
+      end
+
+      schema = %Schema{full_name: schema_name.name}
+      Codec.Plain.encode(payload, schema: schema)
```

Comment:
In `encode/2` this variable was used to avoid duplication, but here it can be inlined

```suggestion
      Codec.Plain.encode(payload, schema: %Schema{full_name: schema_name.name})
```